examples/nlp/intent_slot_classification/conf/intent_slot_classification_config.yaml

```diff
@@ -0,0 +1,91 @@
+# Sentence classification with pretrained BERT models
+
+trainer:
+  gpus: 1 # the number of gpus, 0 for CPU
+  num_nodes: 1
+  max_epochs: 100
+  max_steps: null # precedence over max_epochs
+  accumulate_grad_batches: 1 # accumulates grads every k batches
+  amp_level: O0 # O1/O2 for mixed precision
+  distributed_backend: ddp
+  checkpoint_callback: False  # Provided by exp_manager
+  logger: False  # Provided by exp_manager
+
+model:
+  data_dir: /home/vgetselevich/data/intent_slots/atis/nemo-processed # /path/to/data
+  class_balancing: null # or weighted_loss
+  intent_loss_weight: 0.6 # only if class_balancing = weighted_loss
+  pad_label: '-1'
+  none_slot_label: 'O'
+  ignore_extra_tokens: false
+  ignore_start_end: false
+
+  train_ds:
+    prefix: train
+    batch_size: 32
+    shuffle: true
+    num_samples: -1
+    num_workers: 2
+    drop_last: false
+    pin_memory: false
+
+  validation_ds:
+    prefix: test
+    batch_size: 32
+    shuffle: false
+    num_samples: -1
+    num_workers: 2
+    drop_last: false
+    pin_memory: false
+
+  language_model:
+    pretrained_model_name: bert-base-uncased
+    max_seq_length: 50
+    bert_checkpoint: null
```

Comment:
Please change it to bert_checkpoint_file to be consistent.

Reply:
Ok, I changed it to "bert_checkpoint_file", but in Evelina configs it is still called ""bert_checkpoint"